tests/test_all_fields.py

```diff
@@ -88,13 +88,13 @@ def test_run(self):
 
                 # verify all fields for a stream were replicated
                 self.assertGreater(len(expected_all_keys), len(expected_automatic_keys))
-                self.assertTrue(expected_automatic_keys.issubset(expected_all_keys), msg=f'{expected_automatic_keys-expected_all_keys} is not in "expected_all_keys"')
-
-                if stream == 'abandoned_checkouts':
-                    expected_all_keys.remove('billing_address')
-                elif stream == 'orders':
-                    # No field named 'order_adjustments' present in the 'order' object
-                    #   Documentation: https://shopify.dev/api/admin-rest/2021-10/resources/order#resource_object
-                    expected_all_keys.remove('order_adjustments')
+                self.assertTrue(expected_automatic_keys.issubset(expected_all_keys),
+                                msg=f'{expected_automatic_keys-expected_all_keys} is not in "expected_all_keys"')
+
+                if stream == 'orders':
+                    # No field named 'order_adjustments', 'total_price_usd' present in the 'order' object
+                    # Documentation: https://shopify.dev/api/admin-rest/2021-10/resources/order#resource_object
```

Comment:
Please reference JIRA in comment:
https://jira.talendforge.org/browse/TDL-15985